config/projects.yml

```diff
@@ -260,8 +260,6 @@ google-authenticator:
       php: ['7.1']
     2.x:
       php: ['7.0', '7.1']
-    1.x:
-      php: ['5.3', '5.4', '5.5', '5.6', '7.0', '7.1']
```

Comment:
Uuuuh what?

Reply:
If we go back to the 2 branch road, we have to remove it.

Reply:
Oh right it's a removal